eclipse-collections-code-generator/src/main/resources/test/set/mutable/abstractPrimitiveSetTestCase.stg

```diff
@@ -457,6 +457,16 @@ public abstract class Abstract<name>SetTestCase extends AbstractMutable<name>Col
         Verify.assertInstanceOf(Unmodifiable<name>Set.class, set.asUnmodifiable());
         Assert.assertEquals(new Unmodifiable<name>Set(set), set.asUnmodifiable());
     }
+
+    @Test
+    public void union()
```

Comment:
I think we should add tests for both sets being empty, and for either being empty.

Reply:
Added tests for 1. both sets being empty. 2. left (set1) is empty 3. right (set2) is empty.
Lines: 479-492.